.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,42 @@
+{
+  "image": "mcr.microsoft.com/devcontainers/universal:2",
+  "features": {
+    "ghcr.io/ddev/ddev/install-ddev:latest": {}
+  },
+  "customizations": {
+    "vscode": {
+      "extensions": [
+        "felixfbecker.php-debug",
+        "wongjn.php-sniffer",
+        "neilbrayfield.php-docblocker",
+        "bmewburn.vscode-intelephense-client",
+        "andrewdavidblum.drupal-smart-snippets",
+        "mblode.twig-language-2",
+        "timonwong.shellcheck",
+        "rogalmic.bash-debug"
+      ]
+    }
+  },
+  "portsAttributes": {
+    "8080": {
+      "label": "Backend",
+      "onAutoForward": "openPreview"
+    },
+    "3000": {
+      "label": "Frontend"
+    },
+    "3306": {
+      "label": "Database"
+    },
+    "8027": {
+      "label": "Mailhog"
+    },
+    "8036": {
+      "label": "phpMyAdmin"
```

Comment:
this configures ports for things which are not existing, this creates confusion. We shall fix this before merging it